netperf_tool/netperf_tool/__init__.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from inspect import isclass
+
+import rclpy.qos
+from rpyutils import add_dll_directories_from_env
+
+# Since Python 3.8, on Windows we should ensure DLL directories are explicitly added
+# to the search path.
+# See https://docs.python.org/3/whatsnew/3.8.html#bpo-36085-whatsnew
+with add_dll_directories_from_env('PATH'):
+    from netperf_tool.netperf_tool_impl import (
+        ClientRunner as _ClientRunner,
+        ServerRunner as _ServerRunner,
+    )
```

Comment:
nitpick: import one thing per line

```suggestion
    from netperf_tool.netperf_tool_impl import ClientRunner as _ClientRunner
    from netperf_tool.netperf_tool_impl import ServerRunner as _ServerRunner
```